aerogearcatalog/sync-app-apb
# feat: add dashboard to (grafana) configmap(s)

### Motivation
https://issues.jboss.org/browse/AEROGEAR-8512

### Verification steps
1. Spin up minishift and/or configure ASB on your cluster with `psturc` organisation and tag `AEROGEAR-8512`
1. Provision metrics-apb
1. Provision sync-app-apb with params:
    * docker image: `docker.io/psturc/voyager-server-example-metrics`
    * metrics endpoint: `metrics`
    * sync app port: `4000`
    * rest of params doesn't matter
1. Verify `sync-app-<your-app-name>-dashboard.json` is present in `grafana-dashboards-configmap`
1. Go to Grafana and expand list of dashboards - and Sync app dashboards should be present
1. Rest of the verification should be covered in https://github.com/aerogearcatalog/metrics-apb/pull/92
